### ISSUE TITLE
Feature request: VIM support

### ISSUE THREAD

**Issue by SchumErik** [holzschu/Carnets#93]
There is a VIM extension for Jupyter lab - having this available here would be great

**Comment by holzschu** [comment]
I'm making a note of this one. It is this extension, right: https://github.com/jwkvam/jupyterlab-vim ?
It will have to wait until I activate jupyterlab, of course.

**Comment by SchumErik** [comment]
Yes! Works nicely in Jupyterlab. There should also be a versIon for Jupyter notebooks. Thanks for taking into consideration, would make for a great environment ...
